[WBH] Documentation migration of web hosting classic (#2604)
scaleway/docs-content: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx

You can initiate the migration of your Web Hosting plan directly from your [Dedibox/Online.net console](https://console.online.net/). For further information, refer to our dedicated documentation [How to migrate an Online.net/Dedibox web hosting plan to Scaleway](/managed-services/webhosting/how-to/migrate-webhosting-plan/).

## Web Hosting classic migration -  Most common questions

### What will my new hosting plan be?

After [initiating the migration](/managed-services/webhosting/how-to/migrate-webhosting-plan/) of your Online.net hosting plan, it will be transferred to Scaleway. During this transition, your existing offer will be temporarily moved to a migration plan. You will then have the opportunity to select from Scaleway's range of Web Hosting plans, tailored to meet your specific requirements.

Your current offer will be transferred to one of these migration plans:

| Online.net/Dedibox Web Hosting offer | Scaleway Web Hosting offer    |
|--------------------------------------|-------------------------------|
| Personal (€1.99/month)               | Migration (€2.99/month)       |

> | Pro (€3.99/month)                    | Lite-over-quota (€5.99/month) |

Author: ```suggestion
| Pro (€3.99/month)                    | Lite                     (€5.99/month) |
```

> | Business (€5.99/month)               | Lite-over-quota (€5.99/month) |

Author: ```suggestion
| Business (€5.99/month)               | Lite(€5.99/month)                         |
```

> | Pro (€3.99/month)                    | Lite                     (€5.99/month) |
> | Business (€5.99/month)               | Lite(€5.99/month)                         |

Author: ```suggestion
| Pro (€3.99/month)                    | Lite (€5.99/month)        |
| Business (€5.99/month)               | Lite (€5.99/month)    |
```

---
meta:
  title: Web Hosting Classic migration - Technical information

>   description: This page provides technical information for users having migrated their Web Hosting Classic to cPanel.

Reviewer: ```suggestion
  description: This page provides technical information for users migrating their Web Hosting Classic to cPanel.
```

content:
  h1: Web Hosting Classic migration - Technical information

>   paragraph: This page provides technical information for users having migrated their Web Hosting Classic to cPanel.

Reviewer: ```suggestion
  paragraph: This page provides technical information for users migrating their Web Hosting Classic to cPanel.
```

tags: webhosting classic migration cpanel
dates:
  validation: 2024-01-10
  posted: 2024-01-10
categories:
  - webhosting
---

> You can now migrate your Online.net Web Hosting Classic plan to Scaleway's new Web Hosting platform. This migration brings you a better experience and performance. Benefits of the migration include:

Reviewer: ```suggestion
You can now migrate your Online.net Web Hosting Classic plan to Scaleway's new Web Hosting platform. This migration brings you better performance and an improved user experience. Benefits of the migration include:
```

- **Improved performance:** Migrating to cPanel can result in noticeable performance improvements. Expect more efficient resource utilization and faster loading times for your websites and applications.
- **Up-to-date system:** cPanel's regular updates contribute to an up-to-date and secure hosting environment. This ensures that your system remains resilient in the face of evolving digital challenges.
- **A wide range of features:** Explore a range of features offered by cPanel, covering file management, security options, email functionality, and database management. These features collectively form a powerful toolkit to address diverse hosting requirements.

You can initiate the migration of your Web Hosting plan directly from your [Dedibox/Online.net console](https://console.online.net/). For further information, refer to our dedicated documentation [How to migrate an Online.net/Dedibox web hosting plan to Scaleway](/managed-services/webhosting/how-to/migrate-webhosting-plan/).

<Concept opened>

> ## What will be my new hosting plan?

Reviewer: ```suggestion
## What will my new hosting plan be?
```

> All migrated hosting will adopt the **migration plan**.

Author: ```suggestion
All migrated hosting will transferred to the **migration plan**.
```

> All migrated hosting will transferred to the **migration plan**.

Reviewer: This sentence doesn't make sense to me, but I'm not sure what it should be? Is there a link to the details of the SCW plan, like showing exactly the price and featureset? Also, another question I would have is how does the user migrate?

<Message type="important">
    You are required to update your hosting plan to one of our regular offers once the migration of your data has been completed.
</Message>

</Concept>

<Concept>

## What happens to my backup option?

> If the Web Hosting Classic plan includes the backup option, the same option will be available on Scaleway.

Reviewer: ```suggestion
If the Online.net Web Hosting Classic plan includes the backup option, the same option will be available on Scaleway Web Hosting.
```

</Concept>

<Concept>

## Where can I find my web files?

All web files in the home directory will be moved to the corresponding directory in the new environment.

</Concept>

<Concept>

> Where can I find my main domain files?

Reviewer: ```suggestion
## Where can I find my main domain files?
```

> Your main domain files can be located in the `public_html` or `www` directory.

Reviewer: ```suggestion
Your main domain files should be located in the `public_html` or `www` directory.
```

</Concept>

<Concept>

## Where can I find my subdomain files?

Your subdomain files will be located in the `/subdomain.domain.tld` directory (compared to `/subdomain` in the old environment).

</Concept>

<Concept>

## Where can I find my backup files?

Your backup files will not be directly migrated. You will retain access for one month to retrieve them from the old hosting platform.

</Concept>

<Concept>

## What happens to my FTP accounts?

All FTP accounts and configurations will be migrated. The existing password will remain functional, and the hostname will be updated to `ftp.domain.tld` or the platform's hostname in [Scaleway's Web Hosting Overview](https://console.scaleway.com/webhosting/webhosting).

</Concept>

<Concept>

## What happens to my Web Crons?

Your Web Crons will be migrated and can be found in cPanel under **Cron Jobs**.

</Concept>

<Concept>

## What happens to my logs?

Your log files will not be migrated. New log files can be accessed in cPanel at `~/access-logs` or `~/logs`.

</Concept>

<Concept>

## What happens to my web statistics?

> Your web statistics will be migrated to cPanel. You will find them under Awstats.

Author: ```suggestion
Your web statistics will be migrated to cPanel. You will find them under **Awstats**.
```

</Concept>

<Concept>

> What happens to my web applications?

Reviewer: ```suggestion
## What happens to my web applications?
```

WordPress websites will use WP-Toolkit for application management, while cPanel's Softaculous will be used for installing new web applications.

</Concept>

<Concept>

## What happens to my alias and subdomains?

Your alias domains and subdomains can be managed in cPanel under **Domains**.

</Concept>

<Concept>

## What happens to my emails?

All your emails and mailboxes will be migrated to cPanel.

</Concept>

<Concept>

> ## What happens to my email redirections and Transfers?

Author: ```suggestion
## What happens to my email redirections and transfers?
```

All your email redirections and transfers will be migrated to cPanel.

</Concept>

<Concept>

## What happens to my databases?

All your MySQL and PgSQL databases will be migrated to cPanel. The hostname will change and must be updated in the website's configuration files. The hostname can be set as `localhost` if the script is on the same platform or the platform's hostname or IP address.

</Concept>

<Concept>

> ## What happens to my DNS Configuration (if the nameservers of the domain resolve to ns0.online.net)?

Reviewer: ```suggestion
## What happens to my DNS configuration (if the nameservers of the domain resolve to ns0.online.net)?
```

- **Nameservers:** We will attempt to update the nameservers to `ns1.whm.fr-par.scw.cloud` and `ns2.whm.fr-par.scw.cloud`.

> - **DNS Zone:** Migration of DNS Zone records not conflicting with the new cPanel Zone will be attempted (e.g., user-custom DNS records).

Reviewer: ```suggestion
- **DNS Zone:** Migration of DNS zone records not conflicting with the new cPanel zone will be attempted (e.g., user-custom DNS records).
```

</Concept>

<Concept>

> When will I be billed for the migrated Web Hosting plan?

Reviewer: ```suggestion
## When will I be billed for the migrated Web Hosting plan?
```

> Your Scalewy Web Hosting billing will start either at the expiration date of Classic Hosting plan or upon the change of the offer on Scalaeway, whichever comes first.

Reviewer: This isn't clear to me - what is meant by "upon to change of the offer on Scaleway" - could it be "when you migrate to Scaleway"?

</Concept>

<Concept>

> ## What about PHP Version Compatibility?

Author: ```suggestion
## What about PHP version compatibility?
```
